master/internal/usergroup/postgres_groups.go

```diff
@@ -14,11 +15,53 @@ import (
 
 // AddGroup adds a group to the database. Returns ErrDuplicateRow if a
 // group already exists with the same name or ID.
-func AddGroup(ctx context.Context, group Group) (Group, error) {
-	_, err := db.Bun().NewInsert().Model(&group).Exec(ctx)
+func AddGroup(ctx context.Context, idb bun.IDB, group Group) (Group, error) {
```

Comment:
how about we introduce a rule that db functions that take (context, idb, ...) have `Tx` suffix (i.e. `AddGroupTx`, `AddUsersToGroupTx`, etc.)
```suggestion
func AddGroupTx(ctx context.Context, idb bun.IDB, group Group) (Group, error) {
```

it was item 3 in an early proposal here: https://github.com/determined-ai/ci-shortcuts/pull/2#issuecomment-1070221693

Reply:
No objection here.